presto-plugin-toolkit/src/main/java/com/facebook/presto/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -196,6 +197,9 @@ public void checkCanCreateViewWithSelectFromTable(ConnectorTransactionHandle tra
         if (!checkTablePermission(identity, tableName, SELECT)) {
             denySelectTable(tableName.toString());
         }
+        if (!checkTablePermission(identity, tableName, GRANT_SELECT)) {
```

Comment:
Why `GRANT_SELECT` now is needed? Is this `ANSI` compliant?

Reply:
Yes, creating a view is effectively granting SELECT access on the underlying tables (because the view runs with the owner's permission). Here's a quote from the Oracle documentation:

> "To grant SELECT on a view to another user, either you must own all of the objects underlying the view or you must have been granted the SELECT object privilege WITH GRANT OPTION on all of those underlying objects. This is true even if the grantee already has SELECT privileges on those underlying objects."